decidim-proposals/config/locales/en.yml

```diff
@@ -22,8 +22,10 @@ en:
             comments_enabled: Comments enabled
             geocoding_enabled: Geocoding enabled
             official_proposals_enabled: Official proposals enabled
+            new_proposal_help_text: New proposal help text
```

Comment:
No, I was wondering if the string was autogenerated by `i18n-tasks` or you actually wanted this string. It's OK for me.